src/Saunter/AsyncApiSchema/v2/Identifier.cs

```diff
@@ -19,25 +19,24 @@ public override string ToString()
             return value.ToString();
         }
 
-        public class JsonConverter : Newtonsoft.Json.JsonConverter
+        public class JsonConverter : System.Text.Json.Serialization.JsonConverter<Identifier>
```

Comment:
We need to register this converter by adding the attribute to the Identifier class:

```csharp
[System.Text.Json.Serialization.JsonConverter(typeof(Identifier.JsonConverter))]
public class Identifier
{
    ...
```